src/AppInstallerCLIE2ETests/InstallCommand.cs

```diff
@@ -134,6 +134,27 @@ public void InstallMSIXWithSignatureHashMismatch()
             Assert.False(VerifyTestMsixInstalledAndCleanup());
         }
 
+        [Test]
+        public void InstallExeWithAlternateSourceFailure()
+        {
+            TestCommon.RunAICLICommand("source add", "failSearch \"{ \"\"SearchHR\"\": \"\"0x80070002\"\" }\" Microsoft.Test.Configurable --header \"{}\"");

```

Comment:
>\"\" [](http://example.com/codeflow?start=69&length=4)

I may be missing something but why double quotes for json key and value here, I thought it's regular json

Reply:
The double double-quotes are to convince `cmd` to actually pass in a single double-quote to the process.